<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetClass</class>
 <widget class="QWidget" name="WidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>700</y>
     <width>795</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="Openfile_Btn">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StartBtn">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="FF_Btn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>First Fit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BF_Btn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Best Fit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="WF_Btn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Worst Fit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Clear_Btn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Restore_Btn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Restore</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Save_Btn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>801</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Widget.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>test_clicked()</slot>
 </slots>
</ui>
